make.js

```diff
@@ -1067,7 +899,6 @@ target.b2g = function() {
   echo();
   echo('### Building B2G (Firefox OS App)');
   var B2G_BUILD_CONTENT_DIR = B2G_BUILD_DIR + '/content/';
-  target.bundle();
```

Comment:
we created "bundle" by running generic target, executing will not produce anything different. also we are not pulling pdf.js/pdf.worker.js file from build/ (but from build/generic/build/), so we are okay here.